client/components/password-change/password-change.js

```diff
@@ -0,0 +1,123 @@
+import "./index.css";
+import React from "react";
+import axios from "axios";
+import qs from "qs";
+import PropTypes from "prop-types";
+
+import {passwordConfirmError, passwordChangeApiUrl} from "../../constants";
+
+import getText from "../../utils/get-text";
+
+export default class PasswordChange extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newPassword1: "",
+      newPassword2: "",
+      errors: {},
+    };
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    if (e) e.preventDefault();
+    const {orgSlug} = this.props;
+    const url = passwordChangeApiUrl.replace("{orgSlug}", orgSlug);
+    const {newPassword1, newPassword2} = this.state;
+
+    if (newPassword1 !== newPassword2) {
+      this.setState({
+        errors: {
+          newPassword2: passwordConfirmError,
+        },
+      });
+    }
```

Comment:
@someshkar you have added a check to validate if both the fields have same value and you're updating state but you are not returning the function. This will lead to further execution of the function and request will be made to the API. Please add a return statement to avoid making extra request when the data is invalid.